src/post/post.service.ts

```diff
@@ -452,12 +452,6 @@ export class PostService {
 
         const post = getValue(postResult);
 
-        const existingPost = await this.postRepository.getByApId(post.apId);
-
-        if (existingPost) {
-            return error('post-already-exists');
```

Comment:
This check is moved to ghost-post.service.ts [here](https://github.com/TryGhost/ActivityPub/pull/1054/files#diff-cdf8e932faf153e8dd07b3ab83ed447c2c99d26ef904946a92340f50e16fcd85R141)